packages/gestalt/src/SegmentedControl/__tests__/SegmentedControl.test.js

```diff
@@ -13,3 +16,21 @@ test('SegmentedControl renders', () => {
   ).toJSON();
   expect(tree).toMatchSnapshot();
 });
+
+test('SegmentedControl renders component items', () => {
+  const wrapper = shallow(
```

Comment:
This could probably be a snapshot test too in my opinion, what do you think?

Reply:
Actually I'm quite unsure about when to use snapshot and when to use dom testing. Always worried that am I over using snapshot?  But for this case, if using snapshot, I would prefer to use shallow snapshot. @christianvuerings any thoughts?

Reply:
I think snapshots work great for UI components like this one. By running a snapshot we isolate the component more to future changes because imagine if one day we added a Text label to SegmentedControl that always appears no matter what the items passed in. In this scenario your current test would fail because the count of Text would bump from 3 to 4. My recommendation is make this a snapshot test but also make the last item a dummy div element instead of an Icon so we can further isolate this test away from other components.

Reply:
sounds good. I'll change it.